package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ember-lazy-video",
   "description": "Lazy loading videos for Ember apps",
-  "version": "0.2.5",
+  "version": "0.3.2",
```

Comment:
Could you please revert the change to `version`? I'd like to control the version numbers when I update the next release